_api-reference/cat/cat-aliases.md

```diff
@@ -43,7 +43,7 @@ GET _cat/aliases
 
 All cat aliases URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameters:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameters:
```

Comment:
The link text should match the title of the destination page or anchor.

https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#formatting-and-organization

Reply:
[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)

Reply:
Agree. But then maintain the information about common parameters on that page. 
"In addition to the common URL parameters outlined on [Compact and aligned text (CAT) API](url), you can specify the following parameters: